### PR TITLE
Issue #81, team 5

### PR DESCRIPTION
**Pull request by premik91**
pre-structured CSS file

### CODE REVIEW

File: piplmesh/frontend/static/piplmesh/css/style.css [hunk 1]
```diff
@@ -78,41 +83,41 @@ ul.errorlist {
...
-    width: 300px;
-    text-align: left;
-}
+        .registration .form_field .form_textbox {
+            width: 300px;
+            text-align: left;
+        }
 
-/* buttons */
+/* buttons -> all buttons should have same design */
```
Comment (mitar) [inline piplmesh/frontend/static/piplmesh/css/style.css:123]: I do not understand this comment?

File: piplmesh/frontend/static/piplmesh/css/style.css [hunk 2]
```diff
@@ -12,6 +12,10 @@ body {
     visibility: hidden;
 }
 
+.shown {
```
Comment (mitar) [inline piplmesh/frontend/static/piplmesh/css/style.css:15]: This was not defined before?

File: piplmesh/frontend/static/piplmesh/css/style.css [hunk 3]
```diff
@@ -78,41 +83,41 @@ ul.errorlist {
...
+    .button ul {
+        list-style-type: none;
+        overflow: hidden;
+    }
 
-.button li {
-    float: left;
-    padding-right: 4px;
-}
+    .button li {
```
Comment (mitar) [inline piplmesh/frontend/static/piplmesh/css/style.css:139]: Shouldn't this be indented one more?

File: piplmesh/frontend/static/piplmesh/css/style.css (hunk 1 above)
Comment (mitar) [inline piplmesh/frontend/static/piplmesh/css/style.css:123]: Why then defining design for class and not directly for HTML elements? If we want all to have the same design?

**Comment by mitar** [comment]
This pull request cannot be automatically merged. Please update to the HEAD from main repository.

File: piplmesh/frontend/static/piplmesh/css/style.css (hunk 1 above)
Comment (premik91) [inline piplmesh/frontend/static/piplmesh/css/style.css:123]: Of course you are right. I don't even know why I added that.

File: piplmesh/frontend/static/piplmesh/css/style.css (hunk 2 above)
Comment (premik91) [inline piplmesh/frontend/static/piplmesh/css/style.css:15]: No I just thought if someone wanted to make a visible element from hidden, he could change the class. But it is much easier to just change the atribute. I will delete.

Comment (mitar) [inline piplmesh/frontend/static/piplmesh/css/style.css:15]: Often you use classes for initial state. Afterwards you simply show and hide with jQuery.

Comment (mitar) [inline piplmesh/frontend/static/piplmesh/css/style.css:15]: Also don't add new features in the middle of some other commit. If you are just fixing the CSS file structure, then just fix that. And if you want to add another class, do another commit for that.

**Comment by premik91** [comment]
Ok I suppose this is it :)

File: piplmesh/frontend/static/piplmesh/css/style.css (hunk 2 above)
Comment (mitar) [inline piplmesh/frontend/static/piplmesh/css/style.css:15]: TODO?

Comment (premik91) [inline piplmesh/frontend/static/piplmesh/css/style.css:15]: https://github.com/frendo11/PiplMesh/commit/ec73cae43a0998f4e3a9e58a9d21956808b491c6
I did fix everything?

Comment (mitar) [inline piplmesh/frontend/static/piplmesh/css/style.css:15]: Have you updated the pull request? Here you see that it is still visible in the diff.
